server/src/main/java/org/elasticsearch/common/settings/Setting.java

```diff
@@ -113,6 +113,11 @@
          */
         NodeScope,
 
+        /**
+         * Setting values equal on all nodes
+         */
+        Consistent,
```

Comment:
What stops a normal Setting instance from using this property?

Reply:
This is handled in `SettingsModule#registerSetting`.